### ISSUE TITLE
Python libsvm 3.33 crashes due to unexpected argument to jit

### ISSUE THREAD

**Issue by nilfm99** [cjlin1/libsvm#218]
See this build: https://github.com/Netflix/vmaf/actions/runs/10284246506/job/28459862846

Sample test failure:
```
____________________ ERROR collecting test/doctest_test.py _____________________
test/doctest_test.py:12: in <module>
    from vmaf.core import quality_runner
vmaf/core/quality_runner.py:9: in <module>
    from libsvm import svmutil
.tox/py311/lib/python3.11/site-packages/libsvm/svmutil.py:2: in <module>
    from .svm import *
.tox/py311/lib/python3.11/site-packages/libsvm/svm.py:144: in <module>
    @jit(nopython=True)
E   TypeError: <lambda>() got an unexpected keyword argument 'nopython'
```

The previous tag, 3.32, does not have this problem. It seems [this commit](https://github.com/cjlin1/libsvm/commit/8265a7f23b797eba53bc08f6bd3c35892e9f7040) is responsible. One solution could be to modify the "fallback" definition of `jit` [here](https://github.com/cjlin1/libsvm/blame/master/python/libsvm/svm.py#L141) to take *args and **kwargs and do nothing with them.

**Comment by advilbn** [comment]
You could also yank the current  version 3.33.0 in pypi, that way no one would need to pin/exclude manually this version.

@nilfm99 , in your PR you could use a version of the decorator using, `functools.wrapps `

```python
try:
    from numba import jit
    jit_enabled = True
except:
    from functools import wraps as functools_wraps
 
    def jit(*args, **kwargs):
        def decorator_jit(func):
            @functools_wraps(func)
            def wrapper_jit(*args, **kwargs):
                return func(*args, **kwargs)
            return wrapper_jit
        return decorator_jit
 
    jit_enabled = False
```

**Comment by nilfm99** [comment]
@advilbn I am not a maintainer of this project, but I agree it should be yanked. As it stands, just importing the python code without `numba` installed breaks.

Regarding the suggested code, I agree it looks better this way and will update it.

**Comment by cjlin1** [comment]
We now see the problem and will do a new release as soon as possible

**Comment by advilbn** [comment]
@cjlin1 , Could you also yanked the version 3.33.0 ? As i mentioned before this prevent users or automatic cicd pipelines to fail installing this version. Yanking gives times to not rush patching but providing a good  user event/issue experience.

NoteL You could yank the package in the pypi website using you account.

Thank in advances, for all you help on this.

**Comment by Sinacam** [comment]
@advilbn Version 3.33.0 has been yanked from PyPI, thanks for the reminder.

**Comment by Sinacam** [comment]
@nilfm99 We are in the process of re-releasing 3.33.0, and we opted to use the original commit 8802a107 in your PR. `functools.wraps` seems to be more complicated than necessary for our purpose because `jit` is never going to be anything other than identity, which doesn't have issues with function attributes. Are there other concerns I missed?

**Comment by advilbn** [comment]
@Sinacam , please check `functools.wraps` documentations, https://docs.python.org/3/library/functools.html#functools.wraps. Using it is more a technicality to avoid loosing the name and docstrings when using decorators. the documentation has an example, that illustrates it.

**Comment by Sinacam** [comment]
@advilbn Yes, I am aware `functools.wraps` forwards function attributes such as name and docstrings. As mentioned, `jit` being the identity doesn't suffer from those problems, there isn't a wrapper to begin with. If there are no other concerns, we will proceed as planned.

**Comment by advilbn** [comment]
@Sinacam , thanks for the explanation and your help on this. No concerns from my end.
